config/api.js

```diff
@@ -31,62 +60,50 @@ module.exports = (app) => {
    */
   app.get('/v1/api/jupiter/alias/:aliasName', async (req, res) => {
     const aliasCheckup = await gravity.getAlias(req.params.aliasName);
-
-    res.send(aliasCheckup);
+    return sendSuccessResponse({req, res, data: aliasCheckup});
   });
 
   /**
    * Get a table associated with a user
    */
   app.get('/v1/api/users/:id/:tableName', (req, res, next) => {
-    const { user } = req;
     const { tableName } = req.params;
     const exceptions = ['users'];
-    let model = '';
-
-    logger.info(req.user);
-
     // If table in route is in the exception list, then it goes lower in the route list
     if (exceptions.includes(tableName)) {
       next();
     } else {
-      find.fileSync(/\.js$/, './models').forEach((file) => {
-        const modelName = file.replace('models/', '').replace('.js', '');
-        let isIncluded = tableName.includes(modelName);
-        if (tableName.includes('_')) {
-          if (!modelName.includes('_')) {
-            isIncluded = false;
-          }
-        }
-        if (isIncluded) {
-          model = modelName;
+      const fileFullPath = getModelFilePath(tableName);
+      if(!fileFullPath) {
+        const error = {
+          errors: `couldn't find the table ${ tableName }`
         }
-      });
-
-      const file = `../models/${model}.js`;
-
-      const Record = require(file);
+        return sendFailureResponse({req, res, error});
+      }
+      const { id, publicKey, accountData } = req.user;   
```

Comment:
You will find several places where it is not validated that the objects come in the request, it would be important to add a validation and if the object does not come propagate an exception, to avoid uncontrolled errors in the rest of the flow.

Reply:
Yes. I noticed it but it was not added before. So i thought it will be available always. I'll take care of validation as well.